projects/aca-content/src/lib/components/header-actions/header-actions.component.scss

```diff
@@ -0,0 +1,47 @@
+.aca-page-layout-header {
+
+  .app-search-input {
+    background: none;
+  }
+
+  .action-bar {
+    display: flex;
+    flex: auto;
+    height: 32px;
+    margin-left: 24px;
+
+    adf-toolbar-divider {
+      width: 24px;
+      height: 32px;
+      margin: 4px 0 0 12px;
+    }
+
+    & > button {
+      margin-left: 12px;
+    }
+  }
+
+  .aca-mat-button {
```

Comment:
this needs to go, something in the parent elements messes up the button layout; we should be using just material button here without custom styles